src/style.css

```diff
@@ -1,3 +1,68 @@
+/* CSS variables */
+:root {
+  --active-text-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  margin: 0;
+}
+
+.logo-header {
+  line-height: 0;
+}
+
+.container {
+  display: flex;
+  margin: 0 auto;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  min-width: 780px;
+}
+
+.navigation-list {
+  display: flex;
   margin: 0;
```

Comment:
Don't duplicate code.

Reply:
I didn’t understand what this was about. If I remove this margin, then show the standard from the browser from ul. No matter how I tried it, I can’t do without it))
Other comments have been corrected!